onebusaway-android/src/main/java/org/onebusaway/android/ui/HomeActivity.java

```diff
@@ -1579,14 +1582,54 @@ public void onClick(View arg0) {
         }
     }
 
-    private void checkDisplayZoomControls() {
+    private void checkDisplayZoomControls(boolean leftHandMode) {
         boolean displayZoom = Application.getPrefs().getBoolean(
                 getString(R.string.preference_key_show_zoom_controls), false);
 
-        mMapFragment.displayZoomControl(displayZoom);
+        LinearLayout zoomButtonsLayout = (LinearLayout) findViewById(R.id.zoom_buttons_layout);
+
+        if(displayZoom) {
+            zoomButtonsLayout.setVisibility(LinearLayout.VISIBLE);
+        } else {
+            zoomButtonsLayout.setVisibility(LinearLayout.GONE);
+        }
+
+        RelativeLayout.LayoutParams layoutParams = (RelativeLayout.LayoutParams) zoomButtonsLayout
+                .getLayoutParams();
+
+
+        int marginDp = (int) getResources().getDimension(R.dimen.zoom_btn_layout_margin);
+
+        if(leftHandMode) {
```

Comment:
If there a reason we shouldn't move this IF statement to the `checkLeftHandMode()` method so it's consistent with the FABs?

Reply:
Also, here and elsewhere please use a space between the `if` and `(` - should be `if (`